packages/phone-number-privacy/signer/src/common/error-utils.ts

```diff
@@ -1,4 +1,4 @@
-import { Response } from 'firebase-functions'
```

Comment:
do these work similarly enough that it can just be replaced like this.  it's interesting that this worked with the firebase type even though I ran this as the express app

Reply:
Firebase functions use express under the covers